pkgs/servers/matrix-synapse/default.nix

```diff
@@ -66,7 +67,11 @@ buildPythonApplication rec {
   doCheck = !stdenv.isDarwin;
 
   checkPhase = ''
-    ${lib.optionalString (!enableRedis) "rm -r tests/replication # these tests need the optional dependency 'hiredis'"}
+    ${lib.optionalString (!enableRedis) ''
+      # these tests need the optional dependency 'hiredis'
+      rm -r tests/replication
+      rm -r tests/module_api
```

Comment:
Note I had to remove some more tests that have dependency on redis.

Reply:
They seem rather core to the UX of synapse: https://github.com/matrix-org/synapse/blob/develop/tests/module_api/test_api.py#L52-L80

Isn't there a way to skip those tests that require redis?

Reply:
Removing both of these exceptions seems to have actually worked. I'm not sure why it failed for me 🤷.